lib/include-code-extension.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const toProc = require('./util/to-proc')
+const path = require('path')
```

Comment:
The default path library is os-bound. When working with Antora virtual paths, you should use the posix variant instead.

```
const { posix: path } = require('path')
```

Although not needed here, whenever you use the default path export, you should assign it to the `ospath` constant to make it clear it is os-bound.